build.pkr.hcl

```
@@ -0,0 +1,44 @@
+build {
+  sources = [
+    "source.amazon-ebs.arm64",
+    "source.amazon-ebs.x86_64",
+  ]
+
+  # We are adding this to avoid the failure to install aptitude during build time
+  # Issue #12 has been created for this.
+  provisioner "shell" {
+    inline = ["sudo apt-get update"]
+  }
```

#12 was closed last summer, but this block is still here. Can this block be removed or should the issue be re-opened?

---

Golla claimed this was necessary.  Adding this code was how he resolved #12, I believe.

---

If that is the case, what do you think about rewording the comment a bit to make this slightly less confusing?
 ```suggestion
  # The following provisioner was added to avoid sporadic aptitude install failures
  # at build time.  See issue #12 for details.
  provisioner "shell" {
    inline = ["sudo apt-get update"]
  }
```